src/main/kotlin/com/zepben/evolve/database/sqlite/common/BaseDatabaseWriter.kt

```diff
@@ -168,33 +143,18 @@ abstract class BaseDatabaseWriter(
         }
     }
 
-    private fun postSave(): Boolean =
-        try {
-            if (!writingToExistingFile) {
-                logger.info("Adding indexes...")
-
-                saveConnection.createStatement().use { statement ->
-                    databaseTables.forEachTable { table ->
-                        table.createIndexesSql.forEach { sql ->
-                            statement.execute(sql)
-                        }
-                    }
-                }
-
-                logger.info("Indexes added.")
-            }
-            logger.info("Committing...")
+    private fun postSave(): Boolean {
+        val status = writingToExistingFile || schemaUtils.createIndexes(saveConnection)
```

Comment:
Question: is there a reason we create the indexes after inserting data for the Sqlite writers? If not, we can combine it into `SchemaUtils::createSchema`.

Reply:
It is a massive speed improvement to not have the indexes when inserting data.